tests/Integrations/Laravel/V5_8/QueueTest.php

```diff
@@ -381,11 +381,7 @@ protected function spanProcessOneJob(
             'laravel_queue_test',
             'queue',
             $resourceDetails
-        )->withExactTags([
-            Tag::HTTP_URL => 'http://localhost:9999/queue/workOn',
-            Tag::HTTP_METHOD => 'GET',
-            Tag::HTTP_STATUS_CODE => 200
```

Comment:
Why would these tags be removed? Is it because this is not an entrypoint root span?

Reply:
Yes, exactly

Reply:
It should not have been there in the first place, but it was there as it was a root span, even if it really is a continuation of something else. Only the root span directly tied to the request should have http info.